src/main/java/dk/cngroup/lentils/controller/TeamController.java

```diff
@@ -24,24 +26,30 @@
     private static final String ACTION_TEAM_UPDATE = "/admin/team/update/";
 
     private final TeamService teamService;
+    private final TeamConverter teamConverter;
 
-    public TeamController(final TeamService teamService) {
+    @Autowired
+    public TeamController(final TeamService teamService,
+                          final TeamConverter teamConverter) {
         this.teamService = teamService;
+        this.teamConverter = teamConverter;
     }
 
     @GetMapping
     public String addTeam(final Model model) {
-        fillModelAttributes(model, teamService.getAll(), new Team(), ACTION_TEAM_SAVE);
+        fillModelAttributes(model, teamService.getAll(), new TeamDTO(), ACTION_TEAM_SAVE);
         return VIEW_PATH;
     }
 
     @PostMapping(value = "/add")
-    public String addTeam(@ModelAttribute @Valid final Team team,
+    public String addTeam(@Valid @ModelAttribute("team") final TeamDTO dto,
```

Comment:
OPTIONAL: pojmenoval bych tuto promennou `teamDto`, protoze kdyz to pak pridavas do toho modelu, tak tam davas dto a neni z toho jasne, co za dto to je.